sanic/worker/loader.py

```diff
@@ -103,8 +104,16 @@ class CertLoader:
         "trustme": TrustmeCreator,
     }
 
-    def __init__(self, ssl_data: Dict[str, Union[str, os.PathLike]]):
+    def __init__(
+        self,
+        ssl_data: Optional[
+            Union[SSLContext, Dict[str, Union[str, os.PathLike]]]
+        ],
+    ):
         self._ssl_data = ssl_data
+        self._creator_class = None
```

Comment:
What's the purpose of setting this to `None` here when it's set a few lines below and not used in between?

Reply:
There is a check in `load`. I suppose doing it as a `getattr` down below would work as well.

Reply:
Turns out I was just confusing the early `return` with a `raise`. Now I see why it's possible that it's not always defined.